projects/offender-events-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/offender/Contact.kt

```diff
@@ -18,6 +17,14 @@ class Contact(
     @Column(name = "soft_deleted", columnDefinition = "number", nullable = false)
     @Convert(converter = NumericBooleanConverter::class)
     val softDeleted: Boolean = false,
+
+    @Column(name = "visor_exported")
+    @Convert(converter = YesNoConverter::class)
+    val visorExported: Boolean? = false
```

Comment:
Apologies I missed this in our earlier chats - this should be the `visor_contact` column.
```suggestion
    @Column(name = "visor_contact")
    @Convert(converter = YesNoConverter::class)
    val visorContact: Boolean? = false
```